testdata/recreate/table_golden.cql

```diff
@@ -11,36 +7,39 @@ CREATE TABLE gocqlx_table.loads (
     PRIMARY KEY ((machine, cpu), mtime)
 ) WITH CLUSTERING ORDER BY (mtime DESC)
     AND bloom_filter_fp_chance = 0.01
-    AND caching = {'keys':'ALL','rows_per_partition':'NONE'}
+    AND caching = {'keys': 'ALL', 'rows_per_partition': 'NONE'}
     AND comment = ''
-    AND compaction = {'class':'TimeWindowCompactionStrategy','compaction_window_size':'14','compaction_window_unit':'DAYS'}
-    AND compression = {'sstable_compression':'org.apache.cassandra.io.compress.LZ4Compressor'}
+    AND compaction = {'class': 'TimeWindowCompactionStrategy', 'compaction_window_size': '14', 'compaction_window_unit': 'DAYS'}
+    AND compression = {'sstable_compression': 'org.apache.cassandra.io.compress.LZ4Compressor'}
     AND crc_check_chance = 1
     AND default_time_to_live = 0
     AND gc_grace_seconds = 864000
     AND max_index_interval = 2048
     AND memtable_flush_period_in_ms = 0
     AND min_index_interval = 128
-    AND speculative_retry = '99.0PERCENTILE';
-
+    AND speculative_retry = '99.0PERCENTILE'
+    AND paxos_grace_seconds = 864000
+    AND tombstone_gc = {'mode': 'timeout', 'propagation_delay_in_seconds': '3600'};
 CREATE TABLE gocqlx_table.monkeyspecies (
-    species text PRIMARY KEY,
+    species text,
     average_size int,
     common_name text,
-    population varint
+    population varint,
+    PRIMARY KEY (species)
 ) WITH bloom_filter_fp_chance = 0.01
-    AND caching = {'keys':'ALL','rows_per_partition':'ALL'}
+    AND caching = {'keys': 'ALL', 'rows_per_partition': 'ALL'}
     AND comment = 'Important biological records'
-    AND compaction = {'class':'SizeTieredCompactionStrategy'}
-    AND compression = {'sstable_compression':'org.apache.cassandra.io.compress.LZ4Compressor'}
+    AND compaction = {'class': 'SizeTieredCompactionStrategy'}
+    AND compression = {'sstable_compression': 'org.apache.cassandra.io.compress.LZ4Compressor'}
     AND crc_check_chance = 1
     AND default_time_to_live = 0
     AND gc_grace_seconds = 864000
     AND max_index_interval = 2048
     AND memtable_flush_period_in_ms = 0
     AND min_index_interval = 128
-    AND speculative_retry = '99.0PERCENTILE';
-
+    AND speculative_retry = '99.0PERCENTILE'
+    AND paxos_grace_seconds = 864000
+    AND tombstone_gc = {'mode': 'timeout', 'propagation_delay_in_seconds': '3600'};
```

Comment:
This is just expected result of .ToCQL(), schema is in other cql files, `tcs[id].Input`.
For sake of not polluting this PR, let's have a separate issue on switching from `SimpleStrategy` to `NetworkTopologyStrategy`
Since now we read it from server side result has changed.

Reply:
Are we 100% sure that in tests we would always use the server-side schema description? (i.e. under no circumstances will fallback to client-side description fire up)

Reply:
I will take care of it

Reply:
I have added logic to the test to handle both cases.